modules/ivc_champva/lib/ves_api/configuration.rb

```diff
@@ -10,6 +10,10 @@ def base_path
         'https://dev.ves.va.gov/ves-vfmp-app-svc'
       end
 
+      def service_name
+        'VesApi::Client'
+      end
+
       def connection
         Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
           conn.use :breakers
```

Comment:
Can you add the service name here as well? I think this is needed to get rid of the warning. 
```suggestion
          conn.use(:breakers, service_name:)
```